### ISSUE TITLE
"Rename and Move" only moves the first highlight/annotation

### ISSUE THREAD

**Issue by tzshi** [jlegewie/zotfile#593]
Hi,

After updating to Zotero 6, I noticed that if I want to rename and move a stored file with annotations to a linked file through zotfile, it only moves the first annotation and the rest of the annotations stay in the original file. (Fortunately no annotation was lost :) )

Maybe this is related to @dstillman 's recent update for Zotero 6 compatibility?

Thanks!

**Comment by dstillman** [comment]
What do you mean by "the rest of the annotations stay in the original file"? It's moving the file, so isn't there only one file?

**Comment by dstillman** [comment]
And to be clear, are you talking about Zotero annotations, made in the Zotero PDF reader?

**Comment by tzshi** [comment]
Thanks for the instruction to generate a Debug ID. Here it is: D1426878277

Actually I noticed that after this operation, the original file icon will appear gray and if I double click again (is this syncing from Zotero?), I will see a file with all but the first annotation.

**Comment by tzshi** [comment]
(In the attached debug ID, I used a fresh file with two annotations (highlights) on the first page.)

**Comment by tzshi** [comment]
And I also tried "Rename and Move" a "blank" file without any annotations, and in that case, the renaming was successful and I ended up with only one file.

**Comment by dstillman** [comment]
Does it happen with all other extensions disabled?

**Comment by tzshi** [comment]
Yes, I just tried disabling all other extensions and still the same result.

**Comment by dstillman** [comment]
OK, I can reproduce. Investigating.

**Comment by dstillman** [comment]
OK, sorry, I didn't test ZotFile properly after a recent Zotero change. Should be fixed once the linked PR is merged.

**Comment by tzshi** [comment]
Thank you so much for fixing this so fast!

**Comment by jlegewie** [comment]
I released  a new version (5.1.1) with the patch yesterday so it should be fixed.
